tests/ha_cases/run.sh

```diff
@@ -542,7 +542,7 @@ function test_multi_task_reduce_and_restart_worker() {
 
             echo "try to kill worker port ${worker_ports[$[ $idx - 1 ] ]}"
             ps aux | grep dm-worker${idx} |awk '{print $2}'|xargs kill || true
-            check_port_offline ${worker_ports[$[ $idx - 1] ]} 20
+            run_dm_ctl_with_retry $WORK_DIR 127.0.0.1:$MASTER_PORT2 "list-member --worker --name=worker$idx" '"stage": "offline"' 1
```

Comment:
Use list-member instead of check_port_offline because maybe worker port offline but still running. Restart it below may not working. see [jenkin](https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/dm_ghpr_test/detail/dm_ghpr_test/7816/pipeline)

```
[2020-12-09T02:53:05.224Z] [2020/12/09 10:51:33.777 +08:00] [INFO] [server.go:87] ["starting dm-worker server"]
[2020-12-09T02:53:05.224Z] [2020/12/09 10:52:07.091 +08:00] [INFO] [main.go:84] ["got signal to exit"] [signal=terminated]
[2020-12-09T02:53:05.224Z] [2020/12/09 10:52:08.174 +08:00] [INFO] [server.go:404] ["receive source bound"] [bound="{\"source\":\"mysql-replica-01\",\"worker\":\"worker5\"}"] ["is deleted"=false]
...
[2020-12-09T02:53:05.290Z] [2020/12/09 10:52:12.423 +08:00] [INFO] [main.go:92] ["dm-worker exit"]
```
only one `starting dm-worker server`. Other cases may have same problem, refine it later